edxval/settings/base.py

```diff
@@ -193,3 +193,5 @@
         }
     },
 ]
+
+TTL_CACHE_TIMEOUT = 60 * 60 * 24  # 24 hours
```

Comment:
Would a more descriptive name be better? e.g `TRANSCRIPT_LANG_CACHE_TIMEOUT` etc

Reply:
This timeout seems too large. One hour should be enough imo.

Reply:
Our use of this method is once a day in mgmt command, so I think 24 hours timeout will also work fine here.